### ISSUE TITLE
Cant assing role to user

### ISSUE THREAD

**Issue by mtamagusku** [spatie/laravel-permission#1554]
Hello, I install spatie/laravel-permission in my laravel 7 proyect. When i try to assing a role to an user, i have the next error:
`  SQLSTATE[42S02]: Base table or view not found: 1146 Table 'ab_db.rol_user' doesn't exist (SQL: select * from `rol_user` where `user_id` = 1)`

**Comment by drbyte** [comment]
Yes, it's true. There's no table called `rol_user` in this package.

Please run the migrations, and use the functions provided by the package. This can all be found in the package documentation: https://docs.spatie.be/laravel-permission/v3/introduction/

**Comment by mtamagusku** [comment]
Yes I follow all the steps in that documentation, but the error persist
